Refac sql to util jack

csy-59 opened the pull request.

csy-59 committed:
[Refac] MySql 모듈화 일부 완성

- Ranking 가져오기 제외하고 모듈화 성공

csy-59 committed:
[Feat] Ranking 가져오는 시스템 구축 완료

- GetDataByOrderLimitN 함수 생성

csy-59 committed:
[Refac] 사소한 수정 및 주석 작성

- SetValueByBase >> UpdateValueByBase로 메소드명 변경

csy-59 committed:
[Refac] 주석 추가

Klistas reviewed (approved):
고생하셨습니다.

csy-59 committed:
[Refac] King 피드백 적용 완료

- 기존 스크립트에 MySqlSetting 버전으로 변경 사항 적용
- 다만 Ranking 쪽은 Joker의 피드백 필요 보류  중

strange-tiger reviewed (changes requested):
1. Namespace명이 MySql이라 기존 Namespace와 겹칩니다. 수정해주세요.
2. Enum이 클래스 안에 있어 접근이 힘듭니다. Namespace 내로 빼는 건 어떨까요?
3. (선택 사항) MySql에서 테이블과 컬럼에 관한 데이터를 전달해 주는 것으로 알고 있습니다. 이를 받아 자료구조로 자동으로 업데이트할 수 있도록 하는 편이 Enum으로 하여 변경이 있을 때마다 확장하는 것보다 확장성이 좋으리라 생각됩니다. 

- 수고 많으셨습니다. Namespace 이름만 바꾸면 현재 코드로도 오류는 발생하지 않으니, 그냥 진행하셔도 된다고 생각합니다. 만약 바꾸실 생각이 있으시고 도움이 필요하시다면 얼마든지 돕겠습니다.

csy-59 committed:
[Refac] Joker 피드백 적용

- 기존 namespace MySql > Asset.MySql 로 수정
- Enum들 모두 namespace 단계로 올림
- 기타 메소드 명 변경

strange-tiger reviewed (commented):
- Refactoring

csy-59 committed:
[Refac] 필요 없어진 변수 삭제

- 추가적으로 using 문도 삭제하려 했으나 후의 추가 시 필요할 수도 있기에 남김

csy-59 committed:
[Feat] enum 동적 생성

- tableTypeEnum, ~column Enum 등을 Script/Util에 생성하는 프로그램 작성
